storage/src/main/java/tech/pegasys/teku/storage/client/RecentChainData.java

```diff
@@ -669,6 +669,14 @@ public Bytes32 getProposerHead(final Bytes32 headRoot, final UInt64 slot) {
         || !isProposingOnTime
         || isProposerBoostActive
         || maybeHead.isEmpty()) {
+      LOG.debug(
+          "getProposerHead - return headRoot - isHeadLate {}, isShufflingStable {}, isFinalizationOk {}, isProposingOnTime {}, isProposerBoostActive {}, head.isEmpty {}",
+          () -> isHeadLate,
+          () -> isShufflingStable,
+          () -> isFinalizationOk,
+          () -> isProposingOnTime,
+          () -> isProposerBoostActive,
+          () -> headRoot.isEmpty());
```

Comment:
we don't need lambdas here since values are immediate.

Reply:
when i dont use lamdas you tell me to use lamdas...

Reply:
I believe the issue is that the Logger API does not support a mix of values AND lamdas together. So either we use all values or all lambdas.

AFAIK there is not downside in using lambdas so if it is for the sake of consistency I am ok with it.